src/topology/continuous_function/bounded.lean

```diff
@@ -1030,4 +1031,101 @@ module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
 end normed_algebra
 
+section normed_lattice_ordered_group
+
+variables [topological_space α] [normed_lattice_add_comm_group β]
+
+-- Can we infer this from https://github.com/leanprover-community/mathlib/blob/f29b0b49badab1bcbe338cb79d102e36e79dce09/src/topology/continuous_function/basic.lean#L114 ?
+instance : partial_order (α →ᵇ β) := partial_order.lift (λ f, f.to_fun) (by tidy)
+
+/--
+Continuous normed lattice group valued functions form a meet-semilattice
+-/
+instance : semilattice_inf (α →ᵇ β) :=
+{ inf := λf g,
+  { to_fun := λ t, f(t)⊓g(t),
+    continuous_to_fun := begin
+      continuity,
+    end,
```

Comment:
```suggestion
    continuous_to_fun := f.continuous.inf g.continuous,
```

Reply:
this one is not really a style comment, but 3 lines for this did not feel right. You could also have written `by continuity` on one line.